content/_changelogs/10.7.0.md

```diff
@@ -0,0 +1,81 @@
+## 10.7.0
+
+_Released 8/30/2022_
+
+**Features:**
+
+- Re-introduces the experimental
+  [Cypress Studio feature](https://docs.cypress.io/guides/references/cypress-studio).
+  Cypress Studio provides a visual way to generate tests within Cypress by
+  recording interactions against the application under test. Cypress Studio is
+  available only in end-to-end tests, and must be enabled via the
+  `experimentalStudio` flag. Addresses
+  [#23461](https://github.com/cypress-io/cypress/issues/23461)
+- Adds [Svelte](https://svelte.dev/) component testing support. Addresses
+  [#23431](https://github.com/cypress-io/cypress/issues/23431) and
+  [#23432](https://github.com/cypress-io/cypress/issues/23432).
+- The "Latest Runs" and "Average Duration" columns in the specs list now provide
+  actions when hovered over to assist with the process of connecting the project
+  to the Cypress Dashboard. Addresses
+  [#22932](https://github.com/cypress-io/cypress/issues/22932)
+- New informational banners will display instructions to assist with connecting
+  projects to the Cypress Dashboard and recording a first test run. Users who
+  don't plan to use the Cypress Dashboard can dismiss the banners to not see
+  them in the future. Addresses
+  [#22933](https://github.com/cypress-io/cypress/issues/22933),
+  [#22934](https://github.com/cypress-io/cypress/issues/22934),
+  [#22798](https://github.com/cypress-io/cypress/issues/22798), and
+  [#22935](https://github.com/cypress-io/cypress/issues/22935)
+- Vue.js projects with a custom spec pattern configuration can now use the
+  "Create Spec from Component" capability introduced in v10.5.0. Addresses
+  [#23071](https://github.com/cypress-io/cypress/issues/23071)
+- Angular component testing projects can provide a custom configuration to the
+  Webpack dev server using the `projectConfig` key. This enables support for
```

Comment:
As in a link to the Webpack dev server configuration schema? Maybe this would be better phrased as "can provide configuration customizations to the Webpack dev server"

Reply:
Added link to new doc section